src/main.ts

```diff
@@ -66,33 +75,51 @@ export async function run() {
             core.warning(`Unable to find 'debugSymbols' @ ${debugSymbols}`);
         }
 
-        await pTimeout(
-            runUpload(
-                packageName,
-                track,
-                inAppUpdatePriorityInt,
-                userFractionFloat,
-                whatsNewDir,
-                mappingFile,
-                debugSymbols,
-                releaseName,
-                changesNotSentForReview,
-                existingEditId,
-                status,
-                validatedReleaseFiles
-            ),
-            {
-                milliseconds: 3.6e+6
+        let retries = maxRetriesForUploadInt;
+        let success = false;
+        while (retries >= 0) {
+            try {
+                await pTimeout(
+                    runUpload(
```

Comment:
If this fails part-way through, it will leave a dangling edit. We probably need to do a retry around each request, and try to discard the edit on fail

Reply:
Oufff, maybe I bit off more than I could chew. Not familiar with Google Play's API and how the edits work. Will try to take another stab tomorrow after work, getting late here! Thanks for the code review!

Reply:
No worries! Thanks for taking the time to look at this :)